### DISCUSSION TITLE
Evergreen doesnt exist theme in base theme

### DISCUSSION THREAD

**Discussion by alfrimpong** [twilio-labs/paste#3131] [General] [answered]
https://github.com/twilio-labs/paste/tree/main/packages/paste-customization

I'm attempting to override some things in the evergreen theme, however, evergreen isnt an option for `baseTheme` how can I go about this?

```
<CustomizationProvider
    baseTheme="default"
    theme={{
      "fontSizes": {
          ...
      }
    }}
  >
```

**Comment by SiTaggart** [comment, marked as answer]
Hi @alfrimpong 

It would be useful to know what you are needing to override in the Evergreen theme and for what project.

The very detailed answer to this can be found in this previous discussion thread https://github.com/twilio-labs/paste/discussions/3026.

TL;DR is, we don't plan on supporting the migration themes for very long, so we won't be adding them as options to the customization provider, as this is designed for customers to use to theme their applications built on top of Flex. We don't want to be making unnecessary breaking changes to customer facing APIs.

**Reply by alfrimpong** [reply]
Gotcha. I'm working internally. Changes to the paste version overrode a lot of things and got our UI out of whack. I'll slack you
